WordPress/Classes/ViewRelated/Me/App Settings/DebugMenuViewController.swift

```diff
@@ -39,41 +37,25 @@ class DebugMenuViewController: UITableViewController {
     }
 
     private func reloadViewModel() {
-        let remoteFeatureFlags = RemoteFeatureFlag.allCases.filter({ $0.canOverride })
-        let remoteFeatureFlagsRows: [ImmuTableRow] = remoteFeatureFlags.map({ makeRemoteFeatureFlagsRows(for: $0) })
-
-        let localFeatureFlags = FeatureFlag.allCases.filter({ $0.canOverride })
-        let localFeatureFlagsRows: [ImmuTableRow] = localFeatureFlags.map({ makeLocalFeatureFlagsRow(for: $0) })
-
         handler.viewModel = ImmuTable(sections: [
-            ImmuTableSection(headerText: Strings.remoteFeatureFlags, rows: remoteFeatureFlagsRows),
-            ImmuTableSection(headerText: Strings.localFeatureFlags, rows: localFeatureFlagsRows),
+            ImmuTableSection(headerText: "General", rows: generalRows),
             ImmuTableSection(headerText: Strings.tools, rows: toolsRows),
             ImmuTableSection(headerText: Strings.crashLogging, rows: crashLoggingRows),
             ImmuTableSection(headerText: Strings.reader, rows: readerRows),
         ])
     }
 
-    private func makeLocalFeatureFlagsRow(for flag: FeatureFlag) -> ImmuTableRow {
-        let overridden: String? = overrideStore.isOverridden(flag) ? Strings.overridden : nil
-
-        return SwitchWithSubtitleRow(title: String(describing: flag), value: flag.enabled, subtitle: overridden, onChange: { isOn in
-            try? self.overrideStore.override(flag, withValue: isOn)
-            self.reloadViewModel()
-        })
-    }
+    // MARK: Tools
 
-    private func makeRemoteFeatureFlagsRows(for flag: RemoteFeatureFlag) -> ImmuTableRow {
-        let overridden: String? = overrideStore.isOverridden(flag) ? Strings.overridden : nil
-        let enabled = flag.enabled(using: remoteStore, overrideStore: overrideStore)
-        return SwitchWithSubtitleRow(title: String(describing: flag), value: enabled, subtitle: overridden, onChange: { isOn in
-            try? self.overrideStore.override(flag, withValue: isOn)
-            self.reloadViewModel()
-        })
+    private var generalRows: [ImmuTableRow] {
+        [
+            NavigationItemRow(title: "Feature Flags") { [weak self] _ in
+                let vc = UIHostingController(rootView: DebugFeatureFlagsView())
+                self?.navigationController?.pushViewController(vc, animated: true)
+            }
+        ]
     }
```

Comment:
This screen doesn't reload, so there are no reasons to store it in memory.

Reply:
Fair enough. 👍 

Do you think it's a safe assumption to expect the screen to keep not having to reload?

Thinking in terms of YAGNI and "efficiency", this is the simpler implementation. Thinking in terms of "future proofing" I feel like having it `lazy` from the beginning would avoid having to notice something that doesn't change is computed repeatedly in a future where this screen will start reloading.

Both options make assumptions 🙃 One that the screen won't change, so we should "optimize" for how it is now. The other that it will change, so we should optimize for a general behavior.

Also, the terms "optimize" and "efficiency" are a bit bloated here, given I'm arguing about an array with one element. It's cheap regardless, whether computed a few times or stored in memory.

Neither is perfect, and I don't have a preferred option. Just nerding out. Keen to hear what you think 😄

Reply:
> Do you think it's a safe assumption to expect the screen to keep not having to reload?

I think it's a safe assumption that the debug screen will also be replaced with SwiftUI soon enough 😁 Speaking about SwiftUI, in its context, re-creating values is considered normal, even if it happens on every redraw.

I usually err on the side of avoiding any unnecessary state. If it was obvious from the code that it was an expensive computation, I would consider caching the result. I think it's safe to say that in this case the amount of work is insignificant, so it makes no difference whether it's stored or not.

The other sections are also implemented as computed properties, so consistency is also a factor. I would avoid making any unnecessary changes.